core-designsystem/src/main/java/com/google/samples/apps/nowinandroid/core/designsystem/component/TopAppBar.kt

```diff
@@ -75,9 +73,9 @@ fun NiaTopAppBar(
 fun NiaTopAppBarPreview() {
     NiaTopAppBar(
         titleRes = android.R.string.untitled,
-        navigationIcon = Icons.Default.Search,
+        navigationIcon = NiaIcons.Search,
```

Comment:
This change is correct but I'm surprised it wasn't caught by the custom lint rules... Same with the changes below